packages/engine-server/src/markdown/utilsv5.ts

```diff
@@ -130,6 +112,8 @@ export type ProcDataFullOptsV5 = {
    * frontmatter variables exposed for substitution
    */
   fm?: any;
+  wikiLinksOpts?: WikiLinksOpts;
```

Comment:
It might make more sense to let the proc calculate them itself based on config, dest and flavor. However that change would require a more significant refactor of existing tests. I don't mind doing that refactor if you think it's done better now than later.